src/cmd/cli/command/commands.go

```diff
@@ -706,15 +676,19 @@ func printEndpoints(serviceInfos []*defangv1.ServiceInfo) {
 
 var composeUpCmd = &cobra.Command{
 	Use:         "up",
-	Annotations: authAndProjectNeededAnnotation,
+	Annotations: authNeededAnnotation,
 	Args:        cobra.NoArgs, // TODO: takes optional list of service names
 	Short:       "Like 'start' but immediately tracks the progress of the deployment",
 	RunE: func(cmd *cobra.Command, args []string) error {
 		var force, _ = cmd.Flags().GetBool("force")
 		var detach, _ = cmd.Flags().GetBool("detach")
 
 		since := time.Now()
-		deploy, err := cli.ComposeStart(cmd.Context(), client, project, force)
+		proj, err := client.LoadCompose()
```

Comment:
wonder if we can move this to the `composeCmd`? Would make a lot of sense!

Reply:
Yeah, had the same idea, now moved to the respective Compose command.